### PR TITLE
HDDS-4782.Merge SCMRatisSnapshotInfo and OMRatisSnapshotInfo into a single class

### PR DESCRIPTION
**Pull request by bshashikant**
## What changes were proposed in this pull request?
SCMRatisSnapshotInfo and OMRatisSnapshotInfo seem to be duplicate of each other. The idea is to merge them into a single class and use as is.

## What is the link to the Apache JIRA

https://issues.apache.org/jira/browse/HDDS-4782

## How was this patch tested?
Verified Build, no functional change.

### CODE REVIEW

**Comment by bshashikant** [comment]
cc ~ @GlenGeng @amaliujia

File: hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/OmMetadataManagerImpl.java [hunk 1]
```diff
@@ -47,6 +47,7 @@
 import org.apache.hadoop.ozone.OmUtils;
 import org.apache.hadoop.ozone.OzoneConsts;
 import org.apache.hadoop.ozone.common.BlockGroup;
+import org.apache.hadoop.ozone.common.Checksum;
```
Comment (GlenGeng) [inline hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/OmMetadataManagerImpl.java:50]: Can this pass check style ?

**Review by GlenGeng — COMMENTED** [review]
LGTM, just some typo.

File: hadoop-hdds/common/src/main/java/org/apache/hadoop/ozone/common/ha/ratis/RatisSnapshotInfo.java [hunk 2]
```diff
@@ -34,9 +34,9 @@
  * lastApplied index to this snapshotIndex. OM SnapshotInfo does not contain
  * any files. It is used only to store/ update the last applied index and term.
  */
-public class OMRatisSnapshotInfo implements SnapshotInfo {
+public class RatisSnapshotInfo implements SnapshotInfo {
```
[the remark this answers is not in the document]
Comment (amaliujia) [reply hadoop-hdds/common/src/main/java/org/apache/hadoop/ozone/common/ha/ratis/RatisSnapshotInfo.java:37]: agreed

**Comment by amaliujia** [comment]
This idea overall LGTM

**Review by mukul1987 — APPROVED** [review]

**Pull request merged**